.github/workflows/wp-i18n.yml

```diff
@@ -72,10 +72,11 @@ jobs:
 
       - name: Check if there are file changes
         id: changes
-        uses: UnicornGlobal/has-changes-action@v1.0.12
+        continue-on-error: true
```

Comment:
These changes are less straightforward, but I think this is how we could replace the third party action seamlessly.

The `continue-on-error` will ensure the workflow doesn't stop if the exit code is non-zero (changes exist)`, and `steps.changes.outcome` checks the [outcome of a step before](https://docs.github.com/en/actions/learn-github-actions/contexts#steps-context) `continue-on-error` is applied.